zstackctl/zstackctl/ctl.py

```
@@ -1674,18 +1680,22 @@ def execute_on_peer(self, cmd, useSudo=False):
         scmd = ShellCmd(
             "sudo -u %s ssh -o BatchMode=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null %s -p %s %s \"%s\"" % (
-                self.ssh_exec_user, self.config['peerip'], peer_port, "sudo" if useSudo else "", script))
+                self.ssh_exec_user, peer_host, peer_port, "sudo" if useSudo else "", script))
         scmd(False)
         if scmd.return_code != 0:
             scmd.raise_error()
         return scmd.stdout
 
 
     def scp_to_peer(self, src_path, dst_path):
+        # IPv6 支持：IPv6 地址在 SCP 目标中需要用方括号包裹
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)